cmd/operator-ui/templates/blackducks/index.html

```diff
@@ -54,10 +54,10 @@ <h4>Black Duck Instances</h4>
             <td>
               <form action="<%= blackduckStatePath({ blackduck_id: blackduck.Spec.Namespace+":"+blackduck.Name }) %>" method="post">
                 <input name="authenticity_token" type="hidden" value="<%= authenticity_token %>">
-                <%= if (capitalize(blackduck.Spec.DesiredState) ==  "START") { %>
+                <%= if (blackduck.Status.State ==  "Running") { %>
```

Comment:
`blackduck.Status.State` can be used to know whether the button should be disabled or not. However, I don't think that it should be used to decide whether it's a Stop or Stop button since that field could have a lot of different values (Error, etc) and that's why the condition was `(capitalize(blackduck.Spec.DesiredState) ==  "START")` . 

How about something like this?

```
 <%= if (len(blackduck.Status.State) == 0) { %>
 	<input type="hidden" id="state" name="state" value="STOP">
	<button name="submit" value="submit" class="btn btn-danger" disabled>Stop</button>
 <% } else if (capitalize(blackduck.Spec.DesiredState) ==  "STOP") { %>
 	<input type="hidden" id="state" name="state" value="START">
	<button name="submit" value="submit" class="btn btn-success">Start</button>
<% } else { %>
	<input type="hidden" id="state" name="state" value="STOP">
	<button name="submit" value="submit" class="btn btn-danger">Stop</button>
<% } %>
```

Reply:
Done.